src/Utils/helper_functions.jl

```diff
@@ -450,3 +450,68 @@ end
 function _isupwards(vec)
     return vec[2] > 0 || (vec[2] == 0 && vec[1] > 0)
 end
+
+
+"""
+    read_gen(filename::String)
+
+Read a sequence of polygons stored in vertex representation (gen format).
+
+### Input
+
+- `filename` -- path of the file containing the polygons
+
+### Output
+
+A list of polygons in vertex representation.
+
+### Notes
+
+The `x` and `y` coordinates of each vertex should be separated by an empty space,
+and polygons are separated by empty lines. For example:
+
+```julia
+1.01 1.01
+0.99 1.01
+0.99 0.99
+1.01 0.99
+
+0.908463 1.31047
+0.873089 1.31047
+0.873089 1.28452
+0.908463 1.28452
+
+
+```
```

Comment:
Can we remove one of the empty lines?

Reply:
hmm im 99% sure it needs both empty lines to work properly.

(i didn't want to spend a lot of time adapting this function, but feel free to improve it).